### PR TITLE
[TODO] fix #16

### LINKED ISSUE #16: relative_to_spp (and therefore reloading) doesn't work with symlinks
```
def relative_to_spp(path):
    spp = sublime.packages_path()
    spp_real = casedpath(os.path.realpath(spp))
    for p in [path, casedpath(os.path.realpath(path))]:
        print(('spp_real:v1', spp_real, p))
        for sp in [spp, spp_real]:
            if p.startswith(sp + os.sep):
                return p[len(sp):]
    return None
```

prints:

('spp_real:v1', '/Users/timothee/git_clone/sublime_packages', '/Users/timothee/git_clone/AutomaticPackageReloader/package_reloader.py')

so `relative_to_spp` returns None

even though it shouldn't:
```
sublime.packages_path()
'/Users/timothee/Library/Application Support/Sublime Text 3/Packages'

realpath '/Users/timothee/Library/Application Support/Sublime Text 3/Packages'
/Users/timothee/git_clone/sublime_packages

ls /Users/timothee/git_clone/sublime_packages
AutomaticPackageReloader -> /Users/timothee/git_clone//AutomaticPackageReloader/

```

### PR DESCRIPTION
**Pull request by timotheecour**
* fixes #16

### CODE REVIEW

**Comment by randy3k** [comment]
could you explain why it is necessary?

**Comment by timotheecour** [comment]
the current logic didn't work in my setting, see https://github.com/randy3k/AutomaticPackageReloader/issues/16

'/Users/timothee/Library/Application Support/Sublime Text 3/Packages'
is a symlink to /Users/timothee/git_clone/sublime_packages
 (because I don't like spaces in paths)

and some packages in /Users/timothee/git_clone/sublime_packages are symlinks (because of how I organize my files), eg:

ls /Users/timothee/git_clone/sublime_packages
AutomaticPackageReloader -> /Users/timothee/git_clone//AutomaticPackageReloader/


try it for yourself, the code didn't work with that setup

**Comment by randy3k** [comment]
I am pretty sure your code will break some windows machines without the `casedpath` function.

**Comment by randy3k** [comment]
Well, I might be wrong, `casedpath` may be necessary now. Need more testing.

File: package_reloader.py [hunk 1]
```diff
@@ -19,11 +19,11 @@ def casedpath(path):
...
     spp = sublime.packages_path()
-    spp_real = casedpath(os.path.realpath(spp))
-    for p in [path, casedpath(os.path.realpath(path))]:
-        for sp in [spp, spp_real]:
-            if p.startswith(sp + os.sep):
-                return p[len(sp):]
+    path = os.path.realpath(path)
+    for f in os.listdir(spp):
+        path2 = os.path.realpath(os.path.join(spp, f))
+        if path.startswith(path2+os.sep):
```
Comment (randy3k) [inline package_reloader.py]: please put a space between `path2` and `os.sep`.
```suggestion
        if path.startswith(path2 + os.sep):
```

File: package_reloader.py [hunk 2]
```diff
@@ -19,11 +19,11 @@ def casedpath(path):
...
 def relative_to_spp(path):
     spp = sublime.packages_path()
-    spp_real = casedpath(os.path.realpath(spp))
-    for p in [path, casedpath(os.path.realpath(path))]:
-        for sp in [spp, spp_real]:
-            if p.startswith(sp + os.sep):
-                return p[len(sp):]
+    path = os.path.realpath(path)
+    for f in os.listdir(spp):
+        path2 = os.path.realpath(os.path.join(spp, f))
```
Comment (randy3k) [inline package_reloader.py]: please use a better variable name rather than `path2`. `path2` suggests that it is a variation of `path`.

**Review by randy3k — CHANGES_REQUESTED** [review]
only two minor things, otherwise it's good

File: package_reloader.py (hunk 2 above)
Comment (timotheecour) [reply package_reloader.py]: done

**Comment by timotheecour** [comment]
PTAL

**Comment by randy3k** [comment]
@timotheecour 

This breaks the detection of the package if the package folder is listed in the sidebar but no file is opened.

I have reverted this PR instead of fixing it. Please resubmit a PR for the fix for your issue at your will.

**Pull request merged**